DrivewayDentDeletion/Operators/cicd-webhook-triggers.yaml

```diff
@@ -507,96 +104,184 @@ spec:
     - name: git-source
       type: git
   tasks:
-    - name: ace-api-build
+    - name: build-mq
       taskRef:
-        name: build-push-ace-api-docker-image
+        name: build-push-docker-image
       params:
+        - name: imageName
+          value: "mq-ddd"
         - name: imageTag
           value: "$(params.imageTag)"
```

Comment:
On master we changed this to avoid redeploying mq as redeploying mq causes downtime, so made the imageTag constant. I think we want to do the same for operators

Reply:
Done